man/org.freedesktop.systemd1.xml

```diff
@@ -4843,6 +4847,13 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
       <varname>StateDirectory</varname>, <varname>CacheDirectory</varname> and <varname>LogsDirectory</varname>,
       which will create a symlink of the given name to the respective directory. The messages take an unused
       <varname>flags</varname> parameter, reserved for future backward-compatible changes.</para>
+
+      <para><varname>ExtraFileDescriptorNames</varname> contains file descriptor names passed to the service via
+      the <varname>ExtraFileDescriptors</varname> property in the <function>StartTransientUnit()</function>
+      method. See <citerefentry><refentrytitle>sd_listen_fds</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+      for more details on how to retrieve these file descriptors. While <varname>ExtraFileDescriptors</varname>
```

Comment:
You mention the `ExtraFileDescriptors` property here but that's not actually implemented?

Reply:
I guess my confusion is that this reads like `ExtraFileDescriptors` is a property I can fetch, but it's only something you can pass to `StartTransientUnit()` at the moment

Reply:
Yea I'm trying to say "while the input to StartTransientUnit is (FD, name) pairs, the output property is only names"

Any recommendations on how to say that more clearly?

Reply:
Talked on IM. Vote is for:
```
Unlike the <varname>ExtraFileDescriptors</varname>
      input property, <varname>ExtraFileDescriptorNames</varname>
      only contains names and not the file descriptors.</para>
```